Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN golint `go list ./cmd/...` && \
 # build
 RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-s -w" -a -installsuffix cgo -o /usr/local/bin/radix-acr-cleanup ./cmd/acr-cleanup/.
 
-FROM mcr.microsoft.com/azure-cli:2.2.0
+FROM golang:1.17.6-alpine3.15
```

Comment:
We must use the `mcr.microsoft.com/azure-cli` image to be able to exexute `az cli` commands used in acr.go.
Version 2.2.0 is a bit old, so it should be updated to a newer version. Latest is 2.9.1

Reply:
Fixed